src/sidebar/config/test/build-settings-test.js

```diff
@@ -39,28 +39,28 @@ describe('sidebar/config/fetch-config', () => {
     $imports.$restore();
   });
 
-  describe('config/fetch-config', () => {
+  describe('#buildSettings', () => {
```

Comment:
```suggestion
  describe('buildSettings', () => {
```

`buildSettings` is a function rather than a method.